test_communication/CMakeLists.txt

```diff
@@ -315,13 +315,19 @@ if(BUILD_TESTING)
   endif()
 
   macro(serialize)
+    if(${rmw_implementation} STREQUAL "rmw_connext_dynamic_cpp" OR ${rmw_implementation} STREQUAL "rmw_opensplice_cpp")
+      message(STATUS "skipping serialize tests for ${rmw_implementation}")
+      set(SKIP_TEST "SKIP_TEST")
```

Comment:
this should be set to an empty string by default, otherwise the variable will keep the same value and the following tests will be skipped as well (same in the other macro below)

Reply:
done in f6ca0ed